runtime/oti/ContinuationHelpers.hpp

```diff
@@ -323,6 +323,65 @@ class VM_ContinuationHelpers {
 
 		return foundInList;
 	}
+
+	/**
+	 * Logic to notify virtual threads waiting on an object monitor.
+	 *
+	 * @param[in] vmThread the J9VMThread
+	 * @param[in] objectMonitor the object monitor on which the virtual threads are waiting
+	 * @param[in] notifyAll indicates if all virtual threads should be notified
+	 *
+	 * @return true if any virtual threads are notified, otherwise false
+	 */
+	static VMINLINE bool
+	notifyVirtualThread(J9VMThread *vmThread, J9ObjectMonitor *objectMonitor, bool notifyAll)
+	{
+		bool notified = false;
+		J9JavaVM *vm = vmThread->javaVM;
+
+		omrthread_monitor_enter(vm->blockedVirtualThreadsMutex);
+		J9VMContinuation **link = &objectMonitor->waitingContinuations;
+		J9VMContinuation *current = NULL;
+
+		while (NULL != *link) {
+			current = *link;
+			if (J9VMJAVALANGTHREAD_DEADINTERRUPT(vmThread, current->vthread)) {
+				/* Remove virtual threads that have been interrupted. */
+				*link = current->nextWaitingContinuation;
+				current->nextWaitingContinuation = NULL;
+			} else {
+				/* Set the notified and onWaitingList flag for vthread that have not been interrupted. */
```

Comment:
Please update this comment: this sets two flags, and  should say "virtual threads" for consistency.